setup.json

```diff
@@ -84,15 +80,12 @@
       "markupsafe==1.1.1",
       "pygments==2.4.2",
       "sphinx-rtd-theme==0.4.3",
-      "sphinx==1.8.5; python_version<'3'",
-      "sphinx==2.2.0; python_version>='3.0'",
-      "sphinxcontrib-contentui==0.2.2; python_version<'3'",
-      "sphinxcontrib-contentui==0.2.4; python_version>='3.0'",
-      "sphinxcontrib-details-directive==0.1.0; python_version>='3.0'"
+      "sphinx==2.2.0",
+      "sphinxcontrib-contentui==0.2.4",
+      "sphinxcontrib-details-directive==0.1.0"
     ],
     "atomic_tools": [
-      "PyCifRW==4.2.1; python_version < '3'",
-      "PyCifRW==4.4.1; python_version >= '3'",
+      "PyCifRW==4.4.1",
       "ase==3.17.0",
       "monty==2.0.4",
       "pymatgen<=2018.12.12",
```

Comment:
I think for ``pymatgen`` and ``monty`` the main reason to use the pre-2019 version was also py2 compatibility.

Reply:
Although we can of course update these later as well.

Reply:
I was leaving all those dependencies to be dealt with in #3547 separately. We intend to relax the pinning as much as possible but that will require quite some testing. For now I just removed those that are explicitly for python<3.5 since that is no longer supported